docker-compose-importer.yml

```diff
@@ -47,19 +47,19 @@ services:
 
   cron:
     #
-    # To make this work, set STATIC_CRON_TOKEN in your .env file or as an environment variable and replace PLEASE_REPLACE_WITH_32_CHAR_CODE below
+    # To make this work, set STATIC_CRON_TOKEN in your .env file or as an environment variable
     # The STATIC_CRON_TOKEN must be *exactly* 32 characters long
     # Use this URL for inspiration: https://www.random.org/strings/?num=1&len=32&digits=on&upperalpha=on&loweralpha=on&unique=on&format=html&rnd=new
     #
     image: alpine
     container_name: firefly_iii_cron
     restart: always
```

Comment:
I just realized the cron entry still needs a `env_file: .env` to function.

Reply:
@PrplHaz4 as @JC5 said indeed that's possible, but you would be repeating the secrets in a file that is supposed to be able to be shareable and not have any sensitive information.
Ideally we'd use [secrets](https://docs.docker.com/compose/how-tos/use-secrets/), but that's for another PR.

@JC5 I just committed the inclusion of the `env_file` for the `docker-compose-importer.yaml`.
I was thinking that it may be a good idea to include the `docker-compose.yaml` from the `docker-compose-importer.yaml` to keep the docker compose DRY and avoid splitbrain issues like this lack of `env_file` in one of the `docker-compose` files.
I can do another PR after this one to propose this change and avoid deviating too much from the original purpose of this PR.